core/services/chainlink/config_test.go

```diff
@@ -1205,6 +1205,17 @@ func TestConfig_full(t *testing.T) {
+			got.EVM[c].ChainWriter.ABI = &abi
+		}
```

Comment:
Can't some/all of these be set from the toml input instead?

Reply:
let me see if I can. That would preferable to doing it this way, for sure.